content/docs/1.5.0/deploy/important-notes/index.md

```diff
@@ -71,5 +71,10 @@ The `Custom mkfs.ext4 parameters` setting was deprecated in Longhorn `v1.4.0` an
 Fast replica rebuilding is supported by Longhorn, and is enabled by default. The feature relies on the change timestamps and checksums of snapshot disk files, so `snapshot-data-integrity` is also set to `fast-check`. The file checksums for snapshot disks will be calculated periodically, with a default check period of 7 days. For more information, please refer to [Fast Replica Rebuild](../../advanced-resources/fast-replica-rebuild/index.html) and [Snapshot Data Integrity Check](../../advanced-resources/snapshot-data-integrity-check/index.html).
 
 ### Each Kubernetes Node Must Have a Unique Hostname for RWX Volumes
+
 Longhorn has a dedicated recovery backend service for NFS servers in the share-manager pods used by the RWX volumes. The clients' information, including its hostname, will be stored in the recovery backend. The information will be used for connection recovery if the share-manager pod is abnormally terminated and a new one is created. The [environment check script](https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-version >}}/scripts/environment_check.sh) helps users to check all nodes have unique hostnames.
 More information please refer to [ReadWriteMany (RWX) Volume](../../advanced-resources/rwx-workloads/index.html).
+
+### Instance Managers Consolidated
+
+Engine instance mangers and replica instance managers has been consolidated. Previous engine/replica instance managers are now deprecated, but they will still provide service to the existing attached volumes.
```

Comment:
Also mentioned `Guaranteed Instance Manager CPU` has been introduced, and the old settings have been removed. (no need to deprecate, because this is a breaking change)